app-modules/notification/src/Notifications/Channels/SmsChannel.php

```diff
@@ -145,27 +186,68 @@ public function send(object $notifiable, Notification $notification): void
                             : NotificationDeliveryStatus::Dispatched,
                         'quota_usage' => $this->determineQuotaUsage($result, $deliverable),
                     ]);
+
+                    if ($smsMessage) {
+                        $smsMessage->quota_usage = $this->determineQuotaUsage($result, $deliverable);
```

Comment:
We already determine the quota usage above, can we reuse that?

Reply:
Done